modules/op-starting-pipelines.adoc

```diff
@@ -64,4 +64,9 @@ To see Pipelines created using the *Pipeline Builder* in the *Topology* view, cu
 .Pipelines on the Topology view
 image::op_pipeline_topology.png[]
 +
-.. In the *Topology* page, hover to the left of the application to see the status of the Pipeline Run for the application.
+.. In the *Topology* page, hover to the left of the application to see the status of the pipeline run for the application.
++
+[NOTE]
+====
+If a pipeline run has failed on a specific task run, on the side panel of the application node in the *Topology* page, the *Pipeline Runs* section under the *Resources* tab displays a log snippet of the failed pipeline run for the application. The log snippet provides a general error message and a snippet of the log, along with a link to the *Logs* section to quickly access the details about the failed run.
```

Comment:
You could split the first sentence into two for readability and to help with localisation. For example:

"You can view a log snippet when a pipeline run fails on a specific task. The log snippets are displayed in the side panel of the application node in **Resources** -> **Pipeline Runs** on the **Topology** page."

Reply:
You could split the last sentence into two, to help with localisation. For example:

"The log snippet provides a general error message and a snippet of the log. A link to the **Logs** section provides quick access to the details about the failed run."

Reply:
Agree with the 2nd suggestion. However, with the 1st suggestion, the SMEs clarified that they prefer the existing draft as it makes more sense. In this instance, it is not really a breadcrumb that a user/developer has to navigate through, hence I did not choose the menu-cascade option. So I will be ignoring the 1st suggestion. Thanks for the understanding :)

Reply:
Ok thank you for clarifying that.